src/main/webapp/index.html

```diff
@@ -12,7 +12,13 @@
         <div id="content">
             <br></br><br></br><br></br>
             <div class="title">houseHOLD</div>
-            <h3 class="body-font">Please log in here:</h3>
+                <h3 class="body-font">Please log in here:</h3>    
+            </div>
+            <div class="login-button">
+                <form action="/login" method="GET">
```

Comment:
do you actually need the form action?

I think that only applies if there is a submit, right?